lib/apis/directUploads.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const generate = (url, payload, config) => axios.post(url, payload, config);
+
+const create = (url, file, config) =>
+  axios.put(url, file, {
+    ...config,
+    transformResponseCase: false,
```

Comment:
It will be best if we can keep the transformations enabled and change the new code accordingly. In future, we intend to have the transformation enabled for all the requests. It will be easier if we do this by default for all new changes. let me know if there is any issues in doing this here.

Reply:
I had a feeling I'd see this review. I had doubts whether this would make the editor should be made too closely coupled with only Neeto projects. Will make this change.